src/demoAskScriptServer.ts

```diff
@@ -0,0 +1,152 @@
+import bodyParser from 'body-parser';
```

Comment:
as I understand we won't need to add this once https://github.com/xFAANG/askql/issues/191 is resolved, correct?

Reply:
Correct.  We can easily merge this PR and then later replace this file with the express code when it works.

Reply:
Can you create a issue for it please so we know that this needs a follow up?

Reply:
Seems like we can do on it already

Reply:
Alas, not yet.  https://github.com/xFAANG/askql/pull/189#issuecomment-648822434